services/QuillLMS/app/views/cms/users/_form.html.erb

```diff
@@ -39,14 +39,19 @@
         <%= f.select :admin_sub_role, [nil].concat(AdminInfo::SUB_ROLES) %>
       </div>
       <div class='cms-form-row'>
-        <%= f.label :admin_approval_status %>
+        <%= f.label 'Admin verification status' %>
         <%= f.text_field :admin_approval_status, :readonly => true %>
       </div>
+    <% elsif @user.teacher? %>
       <div class='cms-form-row'>
-        <%= f.label :email_verification_status %>
-        <%= f.select :email_verification_status, [nil, UserEmailVerification::PENDING, UserEmailVerification::VERIFIED] %>
+        <%= f.label :teacher_verification_status %>
+        <%= f.select :teacher_verification_status, [nil].concat(TeacherInfo::VERIFICATION_STATUSES) %>
```

Comment:
It doesn't matter for this case, but I've had unexpected behavior with `concat`.  Specifically, `concat` modifies the array that it's called on.  In this case, that's a newly instantiated array `[nil]` every time, but in the past when I've done things like `a.concat(b)`, where `a` is kept in memory, subsequent calls re-appended `b` to the newly-combined value.  Eventually you end up with something that looks like `a + b + b + b + b + b + ...`

All of this is a long-winded way of making a take-or-leave comment that I've generally come to prefer `a + b` for array combinations, or in your case here: `[nil] + TeacherInfo::VERIFICATION_STATUSES`.